README.md

```diff
@@ -26,11 +26,14 @@ A curated list of awesome things and projects built by Algerian developpers.
   - [Phyber](https://github.com/ghousseyn/phiber) - Build light-weight lightning-fast PhP applications with Phiber
 - Go
   - [Go Arabic](https://github.com/01walid/goarabic) - A Go Lang package for dealing with Arabic text
+- C
+  - [ArduinoSpritzCipher](https://github.com/abderraouf-adjal/ArduinoSpritzCipher) - Spritz library for Arduino, CSPRNG, cryptographic hash and MAC functions, symmetric-key data encryption, and general-purpose functions
```

Comment:
Hello,
No, I cannot do any change because the projects are archived (Until bug get discovered).